src/common/parseDataUtil.js

```diff
@@ -23,13 +19,40 @@ const parseDataUtil = {
       to: txObject.get('to'),
       confirmations: txObject.get('confirmations'),
       memo: txObject.get('memo'),
-      status,
-      dateTime,
+      confirmedAt: txObject.get('confirmedAt'),
+      createdAt: txObject.get('createdAt'),
+      status: txObject.get('status'),
       objectId: txObject.id,
     };
     return transaction;
   },
 
+  /**
+   * processTransaction, process transaction with transaction, isSender
+   * @param {object} transaction
+   * @param {boolean} isSender
+   */
+  processTransaction(transaction, isSender) {
+    const newTransaction = transaction;
+    const dateTime = newTransaction.status === TxStatus.SUCCESS ? transaction.confirmedAt : transaction.createdAt;
```

Comment:
Transaction can fail. When if fails we need to use the blockchain time as well. Is it called confirmedAt? Find one failed transaction in explorer and you will see the timestamp.